src/components/dialogs/dialogAlert/dialogAlertContent/dialogAlertContent.tsx

```diff
@@ -0,0 +1,14 @@
+import classNames from 'classnames';
+import type React from 'react';
+import { type ComponentPropsWithoutRef } from 'react';
+
+export interface IDialogAlertContentProps extends ComponentPropsWithoutRef<'div'> {}
+
+/**
+ * `DialogAlert.Content` component.
+ */
+export const DialogAlertContent: React.FC<IDialogAlertContentProps> = ({ className, ...otherProps }) => {
+    return <div className={classNames('overflow-auto px-4 md:px-6', className)} {...otherProps} />;
```

Comment:
Don't we need to add a background color to the dialog content/header/footers?
We should also increase the z-index of the overlay/dialog-root
<img width="366" alt="Screenshot 2024-02-23 at 11 29 16" src="https://github.com/aragon/ods/assets/19923822/3a2a0f19-cf9f-4af5-b036-2dd9e56e63e3">

Reply:
The header and footer has the gradient background, remember?